components/LoginForm.js

```diff
@@ -1,12 +1,35 @@
+import axios from '../utils/axios';
+import { useAuth } from '../context/auth'
+import {toast} from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
+import { useEffect } from 'react'
+
+toast.configure()
 export default function RegisterForm() {
-  const login = () => {
-    /***
-     * @todo Complete this function.
-     * @todo 1. Write code for form validation.
-     * @todo 2. Fetch the auth token from backend and login the user.
-     * @todo 3. Set the token in the context (See context/auth.js)
-     */
-  }
+  const { login,API_BASE_URL,setpagetype } = useAuth()
+  useEffect(()=>{setpagetype("LOGIN");return;},[])
+  const loginnow = () => {
+    const user_name = document.getElementById("inputUsername").value.trim();
```

Comment:
Use React Hooks like useState instead of Vanilla JS

Reply:
I have updated login form. Please. review now.